### ISSUE TITLE
Hovering buttons will overlap - Should the actually hovered item "lie above the others"/z-index?

### ISSUE THREAD

**Issue by langz** [dnbexperience/eufemia#2118]
https://user-images.githubusercontent.com/1359205/226293623-77397780-6084-4a9e-8cd6-589804906e71.mov

**Comment by tujoworker** [comment]
Ideally they should have spacing between each other. It would be more realistic I would say. Or do you see other cases, where we should solve it component-wise?

**Comment by langz** [comment]
I agree, spacing sounds good. I'll see if I find any other cases in our examples or stories 🙏

**Comment by langz** [comment]
This example has been updated, hence closing this issue 🙏
